pages/common/az-account.md

```diff
@@ -0,0 +1,24 @@
+# az account
+
+> Manage Azure subscription information.
+> More information: <https://docs.microsoft.com/cli/azure/account>.
+
+- Get a list of subscriptions for the logged in account:
+
+`az account list`
+
+- Set a `subscription` to be the current active subscription:
+
+`az account set --subscription {{00000000-0000-0000-0000-000000000000}}`
+
+- List supported regions for the current active subscription:
+
+`az account list-locations`
+
+- Get an access token to use with `MS Graph API`:
+
+`az account get-access-token --resource-type {{ms-graph}}`
+
+- Get details of current active subscription in `json` format:
+
+`az account show --output json`
```

Comment:
Does this support other types of output formats, if it does, we should list them {{1|2|3}}

Reply:
```suggestion
`az account show --output {{json|tsv|table|yaml}}`
```